org.gnu.Emacs.json

```diff
@@ -9,7 +9,8 @@
         "--share=ipc",
         "--socket=pulseaudio",
         "--socket=x11",
-        "--filesystem=host"
+        "--filesystem=host",
+	"--filesystem=/tmp"
```

Comment:
Would be nice to have spaces for indentation also on this added line.

If you want to, feel free to update the pull request with an adjusted version. Alternatively, I will address it after pulling this in.